tests/unittest/scripts/test_check_test_list.py

```
@@ -0,0 +1,435 @@
+def test_validate_mutated_name_is_unverifiable_not_error(mod, tmp_path):
+    # The soundness payoff: a mutated argvalues const must NOT produce a false
+    # INVALID PARAMETRIZE ID; it becomes unverifiable instead.
+    source = (
+        "import pytest\n"
+        'PARAMS = ["a"]\n'
+        'PARAMS.append("b")\n'
+        '@pytest.mark.parametrize("x", PARAMS)\n'
+        "def test_f(x): pass\n"
+    )
+    lists_dir, base = _make_layout(tmp_path, source, ["test_sample.py::test_f[b]"])
+    errors, unverifiable, accepted, rejected = mod.validate_test_lists(lists_dir, base)
```

Ruff (0.16.1)

[warning] Unpacked variable `accepted` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)